ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestIntegrationTestUtil.java

```diff
@@ -569,7 +570,7 @@ public static List<String> makeWsRequest(
 
       final String uri = baseUri.toString() + "/ws/query?request="
           + buildStreamingRequest(sql, overrides, requestProperties)
-          + "&access_token=" + credentials.get().getAuthHeader();
+          + "&access_token=" + credentials.get().getAuthHeader().replace(" ", "");
```

Comment:
1. what is this change?
2. Do we need to import `URLEncoder`?

Reply:
This was to clean up the url of empty spaces to resolve the url exception